vmb/matrimony/models/profiles.py

```diff
@@ -524,6 +531,27 @@ class MatrimonyProfile(BaseModel):
         null=True,
         help_text="Religious background of the family",
     )
+    father_name = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
```

Comment:
model field's verbose name missing, which is used in the form UI. `verbose_name` should be `Father's name` and similarly for the fields below.